src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -305,42 +305,49 @@ <h5 jhiTranslate="artemisApp.exercise.export.options"></h5>
                                     Online editor
                                 </a>
                             </div>
-                            <div
-                                *ngIf="(newManualResultAllowed || (value.results?.[0]?.assessmentType && value.results[0].assessmentType !== AssessmentType.AUTOMATIC)) &&
-                                        value.results?.[0]?.submission"
-                            >
-                                <a
-                                    [routerLink]="getAssessmentLink(value)"
-                                    class="btn btn-sm me-1"
-                                    [class.btn-success]="(!value.results?.[0]?.assessmentType || value.results?.[0].assessmentType === AssessmentType.AUTOMATIC) && !value.results?.[0]?.hasComplaint"
-                                    [class.btn-primary]="value.results?.[0]?.completionDate || value.results?.[0]?.hasComplaint"
-                                    [class.btn-warning]="!value.results?.[0]?.completionDate && !value.results?.[0]?.hasComplaint"
+                            <ng-container *ngFor="let correctionRound of correctionRoundIndices">
+                                <div
+                                    *ngIf="(correctionRound == 0 || value.results?.[correctionRound - 1]?.completionDate) &&
+                                    (newManualResultAllowed || (value.results?.[correctionRound]?.assessmentType && value.results[correctionRound].assessmentType !== AssessmentType.AUTOMATIC))"
                                 >
-                                    <fa-icon [icon]="faFolderOpen" [fixedWidth]="true"></fa-icon>
-                                    <ng-container *ngIf="!value.results?.[0].hasComplaint; else courseComplaint">
-                                        {{ ('artemisApp.assessment.dashboard.actions.' +
-                                            (!value.results?.[0]?.assessmentType || value.results?.[0].assessmentType === AssessmentType.AUTOMATIC ? 'assess' : (value.results?.[0]?.completionDate ? 'open' : 'continue')))
-                                            | artemisTranslate
+                                    <a
+                                        [routerLink]="getAssessmentLink(value, correctionRound)"
+                                        [queryParams]="this.exercise.exerciseGroup ? { 'correction-round': correctionRound } : {}"
```

Comment:
Previously, the conditional on whether to render the multi or the single variant was `correctionRoundIndices?.length && correctionRoundIndices.length > 1`. Now you check the exerciseGroup.

What's the reason behind this change? Did you do it on purpose?